manim/utils/ipython_magic.py

```diff
@@ -0,0 +1,61 @@
+"""Utilities for using Manim with IPython (in particular: Jupyter notebooks)"""
+
+import mimetypes
+from pathlib import Path
+
+from IPython.core.magic import Magics, magics_class, line_cell_magic, needs_local_scope
+from IPython.display import HTML, Image
+
+from manim import config, tempconfig
+
+from .._config.main_utils import parse_args
+
+
+@magics_class
+class ManimMagic(Magics):
```

Comment:
It could be maybe a bit more obvious that this belongs to jupyter, so maybe renaming it to ManimJupyterMagic?

Reply:
I think if we make it more specific, it should be called `ManimIPythonMagic` -- but I'm not sure that this is actually worth it: after all, there is pretty much no reason whatsoever for a user to interact with this class, or to know about its existence.

Reply:
Sounds also good. I dont mind to much